src/common/EndpointManagement/RequestArguments/EndpointManagerRequestArgument.cs

```diff
@@ -0,0 +1,14 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+namespace Microsoft.BridgeToKubernetes.Common.EndpointManager.RequestArguments
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public abstract class EndpointManagerRequestArgument
```

Comment:
It might be possible but that would require EndpointManagerRequestArgument to be generic like 
`EndpointManagerRequestArgument<T>` so the complexity of the code would increase.

Since some of the request arguments has other properties aswell, I'm not sure if it's worth it.